### ISSUE TITLE
NPE in TextSpinner in LogCacheActivity

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#14890]
For current release 2023.11.23 (but not limited to that release), Play Store reports quite an amount of errors of this type:

```
Exception java.lang.NullPointerException:
  at cgeo.geocaching.log.LogCacheActivity$$ExternalSyntheticLambda15.call (R8$$SyntheticClass)
  at cgeo.geocaching.ui.TextSpinner.lambda$selectTextViewItem$2 (TextSpinner.java:363)
  at cgeo.geocaching.ui.TextSpinner.$r8$lambda$khBvTymG70YgbsW8YlHHxYLOEDY (TextSpinner.java)
  at cgeo.geocaching.ui.TextSpinner$$ExternalSyntheticLambda0.call (R8$$SyntheticClass)
  at cgeo.geocaching.ui.dialog.SimpleDialog.createGroupedDisplayValues (SimpleDialog.java:693)
  at cgeo.geocaching.ui.dialog.SimpleDialog.selectSingleGrouped (SimpleDialog.java:345)
  at cgeo.geocaching.ui.TextSpinner.selectTextViewItem (TextSpinner.java:362)
  at cgeo.geocaching.ui.TextSpinner.lambda$setView$1 (TextSpinner.java:175)
  at cgeo.geocaching.ui.TextSpinner.$r8$lambda$Av7isvhT8nryY0fxn80GK57T_rg (TextSpinner.java)
  at cgeo.geocaching.ui.TextSpinner$$ExternalSyntheticLambda5.onClick (R8$$SyntheticClass)
  at android.view.View.performClick (View.java:7558)
  at com.google.android.material.button.MaterialButton.performClick (MaterialButton.java:1211)
  at android.view.View.performClickInternal (View.java:7534)
  at android.view.View.-$$Nest$mperformClickInternal
  at android.view.View$PerformClick.run (View.java:29661)
  at android.os.Handler.handleCallback (Handler.java:942)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:240)
  at android.os.Looper.loop (Looper.java:351)
  at android.app.ActivityThread.main (ActivityThread.java:8381)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:584)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1013)
```

(It's about 15% of current release's crashes)

**Comment by eddiemuc** [comment]
I will look into it

**Comment by eddiemuc** [comment]
Fixed with PR #14891
